Solutions/moritz_w/Lesson_6/integer_sort.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+
+def insertion_sort(data):
+    sorted_data = [i for i in data]
+    for i in range(1, len(sorted_data)):
+        num = sorted_data[i]
+        j = i
+        while j > 0 and sorted_data[j - 1] > num:
+            sorted_data[j] = sorted_data[j - 1]
+            j -= 1
+        sorted_data[j] = num
+    return sorted_data
+
+
+def merge_sort(data):
+    if len(data) == 1:
+        return data
+    elif len(data) == 0:
+        return []
+
+    new_data = [i for i in data]
+    half_index = int(len(new_data) / 2)
+    left_part = new_data[0:half_index]
```

Comment:
You can use simpler indexing:
left_part = new_data[:half_index]
right_part = new_data[half_index:]
empty first index field means from beginning
empty last index field means till end

You dont have to create new_data = [i for i in data]. Because by creating array slices left_part and right_part you already creating a copy.